src/juju_doctor/builtin/application-exists.py

```diff
@@ -0,0 +1,89 @@
+"""Application-exists verbatim builtin plugin."""
```

Comment:
Add usage example? If we add them here, it would later enable autogen of api docs with sphinx (example: [src](https://github.com/canonical/operator/blob/5410bf006912866a883e1dce60163e87799b9fa6/ops/pebble.py#L15) -> [doc](https://documentation.ubuntu.com/ops/latest/reference/pebble/#module-ops.pebble)).